src/vector/backends/awkward.py

```diff
@@ -1113,11 +1130,10 @@ def longitudinal(self) -> LongitudinalAwkward:
         to access the coordinates.
 
         Examples:
+            >>> import vector
             >>> a = vector.Array(
             ...       [{"px": 1, "py": 1.1, "z": 0.1, "m": 1}, {"px": 2, "py": 2.2, "z": 0.2, "m": 3}],
             ... )
-            >>> a.longitudinal
-            <vector.backends.awkward.LongitudinalAwkwardZ object at 0x0000028231215A80>
```

Comment:
I thought xdoctest recognized the pointers here in an object default repr and still compared correctly? (not sure it's important to show, but just thinking it should have worked if that's the problem)

Reply:
Yes, locally xdoctest fails the doctests on these outputs. I will get them up in the PR closing #209!